app/controllers/blogs_controller.rb

```diff
@@ -9,8 +9,7 @@ def index
 
   # GET /blogs/1
   # GET /blogs/1.json
-  def show
-  end
+  def show; end
```

Comment:
This should be in two lines.

Reply:
if it has no content, it can be in just one line

Reply:
It was corrected